zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnCompensationSubscriptionBehaviour.java

```diff
@@ -38,6 +38,9 @@
 
 public class BpmnCompensationSubscriptionBehaviour {
 
+  /** Default instance key if no compensation handler was activated. */
+  private static final long NONE_COMPENSATION_HANDLER_INSTANCE_KEY = -1L;
```

Comment:
❓ What do you think about moving this static var into the `CompensationSubscriptionRecord` class ?

Reply:
Good question. :thinking: For now, I would keep the static value in the behavior class because it belongs slightly more to the behavior than the protocol itself. However, both would be fine. :+1: